example/controller-registration.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
9LxP8EPmegvy+2EkeLOBjgmKYyhEvPRWHNShEcM4dH1qAJ1V9B5przar72AEEr2G7qSknBUHnsXY+WKX2ZzBEDP+nDkxyu7KY8zztQmU37TMfY83ITMX7cVxKsOmmDpVed5l6nmXISi0+YKSEyFKouzlwvgMfd/SPfAM0lkPQYMYUpu86NDE7izYPVpggoUKsJDBSrawvA3cSA1sjB//Qg+4k6l3/iwLj+aBzfQuILwptbxkypXN+rC1yg3aiZMxtRIjm5Rf1M3JpKYmAKf3BnfenFkekAradJYhJ+qZvN6prDYStUrNOC5DVaVd612ALF8P8regHH8J3YC0n7eLnRXR/g3cleFzi5FPM3XYiioXqkY/q1CEXYhoY7jOC13EyzFyzFUQlK1geKF9p+MnCg5lucymOINiRZiZ5EsnXx1MpmzkO70uDWb6CBLj/GzYPxlevRmeDQcY0ffNef/VcHTZH+iHdNyV9TtQ9ourOL5XU7GhzfOF/3bmP54J4qZTtML39FX/5fAGkL24enNxM7z629XpdQnXHhEmoXabq1N5vWuNWTaz8vQy2tITmFAIfZXXKIYE29tdMcPP8LIGfYVKnZV5VmEjaRTwsZYgf0nVLazMLQc/xVHYa9uQLSvMpRr0Skxqht8CsQSpSvPXMhoJU1MXW5Fzvkbom7VptS6lbHXtUsez9vLjkkuXKkmHiVehAyAOurtaJyRBP/UqdZs+Vlp3/ycKHZiE45RvSo5TZ0JXbwStiv+ze1yK/7B7uI3/8CBJaipG/RneBPQj3Bpuf/OC7JndPWMXFMjAisS2hgua7AdYMcm9IvOlm9zktoS+iRRxOx23kfQAhZUlxmAH7OmxBsXG02XonGRi9i0Xs89sB2rd9aVa6YH5/2MgN5M8KLv3e93U+dSSu00fIq2r/+OxZX/Y+/+7Bwf7Rf3fPdp+/+dhUnH/n/PX0mPiloMAiBhAV6FH30Mnf6b7/TJamYHXcF9iFAAZ10wLmlAKCLBwDGkQGUIL/e5gzh3L3Am/wmDwCAH84R7V/Hs1lBct/OyIU9BGCIioeepJ3ActY1V30lFGSkZZyHIXkWg/a5eBt9tlMNmUviEZq+JcVLCKx7tQmU4W76KmUeEsqRMqp16ZZHUjrVZi4tCjrJgxFpdQRH5eovBqTWTFnXyPVrG6ihttsUxmbfGL4mVe/mMla/G8sQpgfs5chPO++NthGANZllOcD/oC4bQGUbpILl7kfVtR+VmwD4M02k40qjRdtuW6iL5GJCRhmRUV0UwqOJMHKMme11EnpWbFSqDcTsVCs54f6+guXTbea/KT17/+gHMgdF7u7ipmLqFdS4XB0e2G9SjFUu7/w+ddFaBaP976KAvCT22ZPUxa1/5X253rLAFW2P/dw91i/K/9g+42/t+DpMYeA5URAOUw/OhbMyUHEe0UvdbJBqOgGJbn8ZC/hvBUMIDZOJMY8pDd4B8y75Gdv79r42O7V++l8LytYnG0e+3rwWX7t3/sNEYm66ehNnNkq9puDjTeLoQk00/JoFUBtP3bOi3DRGrwiSZrOT/8BTK46FK0sFIg5BG5vji56ImvufK6OEfEoQWmDn4fFCPQ2PxaEHqZBiHxwmBCYxnIQ32s9TbF+C4muaJ+OONf+PTx5iML8ZuiLJOQ7Lz+r7M98/gQv6BCxpQG+CF5lAXpn9qY48KgMMrd5p4VVcKwEfwPJ0CfqRWgNiyFL5uc3E8vIcdWzz1yHgYIXRApW7jx41+kDG9YuAfVk00UkpQDoRxgG4tbog+/Xfufcji24fwvQ140MwNWx/8uzv+H+Ho7/z9AKi6BilO7srA/r8OXB1OCPPwVEmiRXNfhHcWA6pbH6O9bP6w7/meRte72/8rz3+5+8fsv3cOt///DpIL9j+yVX8gs2PxtFHVmg8EIFlK2K4LHt922VBk3eJkLSlyGTl8WrvxIQCOZMwCVhtpDWSIVfVGmnO45upgnVE7m7sozXd1rKHslnIB2nuVXqnw36AszUHfF8cHejec13+0sI2XmUExRtfqDnY1qlj+UKGzczBmv6q4H5pfue+RORU21p9hf06kqcoRfkWbfYU/0wmZe7ncbJ/0/NTXT/zL86rqKX6YV+n//cK94/2uvu/3+y8Mk4Z/PtaL6NmiP0NSc2DFq+Ew8YP2FW7BZRkc/TazSBok16eVxfyPNtf/09jxMLkUM21Zr0b25p+ln9HDuqTOS1iNY8iUYLFcEaJRrRBlfiamNqvGc8AV+fgGlJUsi5MVdrcx/9FEWMkqWzePmuoF2S0td72ipNevXu1/vtlr5wQXeKm2BdsvbcrRGyvPOo9Ic8mhhBtnrfv3KXalZ2zjZtlstzZEdCxWuAvTIIWTWOrgrYxVt5+w6QTY3lgFlh0w94cBf5d3fI/u7BPAqfpu3p9O88vPie/wey8LXlHuyGfWBc0FYQVaelTFRXY4OVdxNQ4b3DG/Fl+dF7HAnYDJ0eMVcJ+ZKLWqD/DD4L+E0gFr0r/StBSYxj80lS6gLPyogMtY2ZrtdE4TN9GgC4m7H8ygxw3jScWBux+2Ludxvq/og9tEuw5246zMu8Cfno909PbRPVlP7OjX8vpCBbXnEQax4RpMdRoaicRWrUtUGAwfTt8OXp+dE/L0a9cnl1elN/3pIfhj+xN9npU2zUHN4flJdA3lc+UlqGfjsOf49MA94weLnorEjKkKvPITHGBOaJDFyP6UBSfEjCFl0MBUweeY6nIWPFro3GF5dn353OgA0VaceqR490rpTVe7DAWsRUvyKvaYs+CAEAGDGong/F5fBeBRVEWBjrg5d1S4rB5b7lLRWfdtaBy6aZSKYs2Fwic1rG9bih7NRF1Z/cq/VKl/M6ZG//wO1dpXvO347ByVI3Lzo8ecF9/z8e30toRGF5rzS5qiJm0zTMY6/fE6qrttwZqvrmwDCp7P2rtk1/wyqVlXutbLva/TQMeJTz+bbtE3btE3btE3N0v8DTA5Z8ACgAAA=
```

Comment:
fix wrong tar (please use gnu tar)
```suggestion
  chart: H4sIAAAAAAAAA+w9a3fbNrL9Wv0KXLU5TtKQshU/dnVvuqvKaurTxPa1XPf0bLYpRMISa5LgEqQcNdv72+8MHhTFh0QpidNsjaYyBQKDwcxgZgAMoAmNXRay2GJvEhYKj4eWmHKeWA6LE0uweOY5rPPZO6VdSEcHB/IvpOJf+bz3dH+ve9A9PMT8vaPdo6PPyMG7NdsspSKhMSGfxdDpVeXWvf9E06QR/+0p8wNvEvKYbdEGMvhwf7+W/8D2Zf5397pP9z8ju++9txXpT87/L8g5TRIWh4IknCgWk9spC8k49XzXCyckos4NnTBht74gl1NPEJFGEY8TeACx8MnE52MS0MSZQuknJGY+TbwZg3rJNJdPQxcAhGwCb3lIHkYxu/beMJfcelDuvx7Z5Cz054SHsiaiRCIWE98Lmd2yj0evRwngBiAGPAgAwNVgRFwvFi174iUd+anQb9nj3+KO/DQZ00kHP8xXMQs7C0Bj6F8akWvPZ6L12Ba3EXyO6Q18JgE8/x8UvaKxx1NBTo6H0GAU81+Zk7Rsz2W0o8pBVsueCYe7rNP62FxtnpqN/8GUxok9p4G/TRvrxn93/6A4/ne7B/fj/y4SjbwrFiPfe2S216JRlH1t79m77ZbLhBN7USKz+uQ7MATEQXEg1zwmyZSR51qEyAgFhwxAcMhICQ7JpMpuhTRgPdJI3lozg8OuDUh8QsPpk0vNxr/LHXvCt21jzfgHW79XGP/gAT69H/93kTodMIPRHCzlNCEPnUcEaP9XMuqfk9GQwACnofxCr8E8ejRhxOFBRMO5Tfpg+mU1ASYfBYW5tvIP0JIS+OuD6IQCLHwaukzpij44E/BnxK+TWwqexgtV5AmZ2aQL2sJhUUKoICFPoB6HKvGtJwBaKKu/OBkMTwExbKHV6cA/A6GikQy21mika++Sh1igrV+1H/03gpjzFPyUOTZKUmgsyTqhEYLWsdtAgBB0mvRXkkUDNsL4ScPg44RCcQoVIvh2nS9IaKKRlmmaJFGv07m9vbWpxNjm8aSjiSY6uq8WYK1r/RCCh4LU/lfqxdDj8ZyAvoYKdAy4+vRWMmwSM3iHzlxIbmNwitD5EprgCMb1RBJ74zRZIprBEbqeLwBkAxFo90fkZNQm3/RHJ6MnCOTHk8vvzn64JD/2Ly76p5cnwxE5uyCDs9Pjk8uTs1P49i3pn/5Evj85PX5CmIecBHKC0wc9ADQ9JCdIDMIaMbaEgjEsImKOd+050LVwkoILSiYcLEMonVIWB55AtgrpWQIY3wu8RDqXotwvuwVFJrw3QWWHciymxHJI27Y78G/GQpfHHXAhp+nYBl53jF5cPEzBTeyY6pbDwyTmvg+aM2YTJJhs2QawqzQpscmXDx2aENXu1fBiBNR6pL+yNxTIwjp1wNEBI0MDvlcG367uJA4PyyM7okOSeQRWuELF17559SpEab3l8Q2LUQBHyv1nbo8kcco6Oxtg30Y2wJRDzii0H8FClF9BlhBXbUhR0JnIdWSow+MYnG+yaIYsNdOK8tDvfYfVqZn9TxhwFvgitloJ2nj9B/z/o4P79Z+7SJvy/zVM+UH3CjuJGs8F1/h/+90S/+HLvf93J+ntW4u47NoLwSvCGVqbWL//3mo2S8O6YDhljVYeEJYC9XztTRQ4y7Ja+YlmGZadNSRsVdM2ONiOz1O3M9ujfjSle60bLwTLM5CFUqXzW+AJpCw+lTPMt28XHlLbvqJ+ygAmtIWuBMiwqmkDrjT1kxNZ08auo+tjarbJRjWhj+DYgNPk6Owt0TBAoM5WyOTqA0rIEu96feULaAcqinMWH9P5/6Y8oVi9/q3s3bZgjdQ0QVAKiuqbwAr5770W0a6DuwaffC1bVylgQh0QHQQYUM/fknkIw5YAtmNerj5gRIicU20rSRKYgvAO2GgAEp2m0pSvH8U8omrB99ILGE8TBauc30ykmsDOcXVbpL0ZFPiezQ2y5nuP/Lu1OZoZtH+Dv+wF8Ac0GAsTsr8NwjCDgqmic4PKTrFHGDRLL9YQtRbUZhg5dLDI0xCW83oSSMJ/wvlLY0AZnbrbMZY6iSR9P02mPPZ+01NDWb3urZrSLDeWe5TPmZnzAphmKAuXITSl4lxubMCsa0q7B4e9xZCT5e2EThZdAOkIk2vSfiD+/kAUS8Ys4sJLeDxfBYL5glUB7G0NsLrbptc+ozC3Zj7MwYBinqsIsMpRsFQVy9TJOw4f2//5s6dN/X+ZrT20pvtBa/d/jo4K/v/+3uHRvf9/F2l5/+dz7V2PmBOz5PNWwBLqUnD4PgfdoDZwVsmJdt11WRFRR/nj9gUMfSqYfWqyQat83sog5+RJlvdCx0/dwkTCBoMwPtxnoWMMKaidj02+Tz5tPP59GDAsvvZ8+BSWzycTL1yjCtaM/7390v7v4f7h/fz/TlJ+/F+DRxAmYy+x1ZPtcTPt7pppt2L/t5L9C/UAY9inY/BE8InkAGWr52p1uS0YzEFqyuiJfqGk0jrG7JiF5YCG4LfEeVlVWQEAtbBuC3cPEB0tqvgI7hGNQaYVloTcsLlaMwAx1lm6gGzQNKLydAG913WM3TbOogxy6ZGbdMzikMEwsR/XIPy4HuHHpWWST5AfMcMtJ2aBQ1nNAF3gkk4ME+LUZ8J8scjOl+iM/ryg5St7RuNXNrDoFdoX3F4DaK/sh3U42F/97TX+X0/qR7jNhyoP3GHm2l9e9p9LVu5U8/I/2cpsqf/VkHg/+n/3YL9b1v/3/t+dpO30zbnSiHl9s9ALJbX5YVSRUS8xm7A3CnjiBWqZBh+ynG95DIMa6j34yXoQWA/cywff9R687D0YPfhNNahAkJ2fsfiz9sO//Q8+fP3Kfbv/uwWfXf15+Y+f2/98/Kj9Snzlsxnzn0FJAQ+g0eZfv7r96tErEYiJBAAD42sbiu78wZXHxuNfeuMBjSy5cjCDGT2PLdyWl3q9UhGsHf+HBf/vKWiAvfvxfxepsI4mmXolmXpmeCrXx5fDBPN7MC9pVKEI8rM4ta9kA5xqoWk2XywoEhpFds5Ig6pa1XBNFS8E7odVrcoNEN0jibR4vSzkhbXgSsItr/QuFtQ+NtNzaePxH7ti00Dg1eO/23169LRg/7tH3fv5352k4pwDnnNbsTd/EcoJMCMeqMWDCyZ4GjvsGFeEPbmiW1YAnt5vQzkq2PGGI7nZJvQ2A7u6jvJbXGs8X65lYplx/1H7HJOYp5Hydsp9k7pMdU1RTW0HyQzfE8n3ucwX8F2+iPw0pn5GNpkHvY2T0wU0nB55C1gCHO/Up3FvkSkcjh5SpjYRHR1LrWFYhLqu5Bn1z3GvgMUD7qdBmLWBZZaDvgcvhzKgO5sDY/pV8PCcJjBVspEq+f3CXCnFydHw4mp4kcvWsWYJIDBZ2exFLrJLbc6uQ6FYSGEwfNk/ebEhAqOEJqnAGM58fFlN+7Ks/MPK3b/sX/4w2rB1JR+y6MomCwVUi5c/nQ83bI+Pf5VRdTFTxEYPGJoIourWzYC3TYXLivIKmf7zMi5unk4FTLT5donLgZehWMdxXczW9Urtn5wOXvxwPDx+fXwGUnCaZ0QUexyd9x7pgiVaTTEFLAtDUZnCmYLALYYODL+wf35y9XRUeEEquesJGdKoxi8ZXNhLuAGwOPGYyENBxZVp66X8Qgs7iIYOfFZbd6otrQ6AvAp3FaMsQwQwOBe8laKYy35dYxiwkhFbKwM8/8VT38Xl+xke+oiZwyeh91sGW54pw0al75AUYErlA2qIzNCFkgfEZAh1zKQkpmEO3kx5WeQlHk/zwmvek+HTotfBoF1jpxweBClYpLmKQcUQZh6LjotztY7wJhaNnSk4Zk6SxqwDhLQk6qHc+rUD94tYWzaxU8C1cgRhkhp+JR9Q3SOjqa6u+rIgt4lrvRiOLolBQLKkyAM1QrOKYsEIJBtQRYY8Y+x6zAMJE9zNiAOdVeCs70GtAlCRjgMvUQEiGKwDHLPJgIYYCz9mJI1wpLo2OQkhN2D+AC3ih2YDUltYSNrmjMh7IOUKinZLr4wtr+WcGqMjKGbGKVYxMf1qzNoFAHWDFpOJMSqmstXTzclngJhwh/uLaHip+SrgrGpbtg9syNvT6lJFfAqVDG7XPp3gxBGBBvDGqTOSy0lrDJAseXIB49yuSciYi0cBqqsoBo45B2+sGrA2AI36c6zKgqJWPpA/N4QF/agBEcjmt3j4IcYA9LHngoNX2yUMUc+Fn2QjqabCOi5hYm+kJasvUOjUUJUneKJG9UE5oRK326nnTPMYroBKFnoAYWWdJw+VcUVdRcM5ag1Nq0croMEQD1b0ElPtcK4qRuOYzmtLafPfmGgn4YchmhGePzzJajWjSSoYssmoGpqoR2l05BeYZsizPuABpPpMD266FM5qbNlFnBGi69B3HJ6GyTdgKaBoM4UGSnVYWV0Z6ZiBGWV4zgswp2TQXzFKtf9CFRhjGaTaVjOhug42UQI3bH5y3FiaZWnDAXjSuAz6+AYeqfIBTM9XSpgmC0IbwwMeJqvrCKZGAglYqKCSC3a9SaeySpkNlhnIJuNcqiIre6SG85T7rgYyD8BfiMFqvewPJIU0vQx9DA1WQjVyIw8Eou8hIe1MA+p8z+Y7eBoywxffCLzAACxevS2R5My6RB4/DlKRPH4sPbHQimBMMfcJGYMPdrhPfrh4AQ4eXjjgytZX8aiJwGGSk5yVJQosOtWB8+DvgQ6UpJDHH7MFXeRSNqbWQCboJBv+Kkd4VacwNdSG+RXmTbunupGfQ6kc3VcpW2s7lhNdeWIAGavYinR7T71cq9FJFgheTwVL6ZLV77Nhua1tqYpWbqK+zytilrOFecUdHYSdNVHbkzAHI8UzktKVwyo4qo9PR/k4dZuofUTyy5SL5JdaoABClujh+cVfnhBmT2zS/ost/2sTgcyHibHJ6R08bWOd9oTzic+sKB37nmO5obCorfLwPCoWqxOSNT5DA9lZ5yssYtjX6vACs0r16tU4ms3aTmgMUPTexaauU28fSrVtpNYaDfZG6mwrVYbZKzuznRprLIYrlIY+EdNE9EbZ0RfsJZrK5t7ZWlTFjReBehhMwddn4YRdUd9zm8+pR7XVSUDjG2HcNUA+t9xfCZYzeVcCrikhCCa1lmMAC7luIwE54DNASfme4aIMEx1cX4vduokqkbMZubyrVSOVtTO05YJdQUsiwREfqUW3n9OvtlJWaScg/660B7HcaI14OaWFI0xLbANfWosTPL3zasyd6FREVGbeK03Tkz+Z0lz5Oq442rluFFSd6tTyF9A3XpAGJEyDMeheULjrFlCyhRO86c2l88XSm7oDZJUGVD3DTYRJ6X39oq/cNGuy7Ks2/8zIyrYC73Lpd6Fj1Fbfpuwl5I212OK2Ih26bKXhTchvQ+vaYzAhzqKYl9P2CvEjIAuDVNDJehK/VOWKbI0Ji2P41GDqca8ZiXys7j96rm8qqfYDljA5K1UxSBlg2b0n8EoJXrUvUmN/rnXUIwyQw/0Nh8971wzUSVLq/4EUhNIE6wUG1YAUF3ODURAl8ydk55zJpZ8deByi6MAD4LNzwag739lYfuTLdahcQqHChjHWs8kgjcGYJvLuTPjYQWWzI52zHYcW989UQs9OmGt8NkS33jezSDH+YQGqNOqrwVhSnlsra6vhsaQJcGUNx/9SXjrOtnMXjWgDQN7enx2TadP4P5dFPp/jSYrmUYBr4n+PdveL9//u7x3s3sf/3UUqx/9FYhHvd5xxuyLCr2l83vuL7W0eEYj3Vdti2pF3gH3oUMKpN5ladAZTUjr2fC+ZmwPSmf6pPsiwOLuUO8ww8xC97zxUafMXeKVdj3TlG3nT3/KdDjpzILdr9KUhvgxCVoSUh5le5Cj7Pmi7DZGMAtFoFWI1ZLiJvoAh09RyCUGkQUdNnZZvFFAEXn1YWN2/ABAMnzA1jHcvIbE497CgT2Uw+xJOD+V1DORL+1L33/4G6IIRbKTd6ChF+9HqrmT3YVT2Tu1xmTtvRk4MIz2c5CvApAUKTVmak4R2BQvbNTUEwsQ6aHjrCqGbocEWEJN3/OWhL/oEX3SuvyS/70eCtxvomKA4XoWIh96Kg5pbMYxDL2AW0Nnc3qP91Wb1XbyBJF8jH6RkghUHPhXidLnLYi5giFl/zYIYdXf1NubpxgRanLRcxB5vQ2Yp2svjVF+bYuepKvPOU98/56DQ5ktKTl1REmUvl8YnDwKaj8CzSGczBC1iaW3yrMMSp7Pk9+QuJliqABMZrOQoz9vChdTQwfvjn+Uv3MnUu3zWhUfz0BH5LiC8KaN+MpXKZnPYucoN2omTMaOJlRnlZ3U2mdTUBODs1pLBmzPqA6mgTXcVcqqeLeud6GojVavUjOsJVFW5Y71LkPXrweItKMdfuReS9pN2sbPqtn8LV2WkbbEWZqYOW1XlzNToZxWKsAs32lie+ywv4uU7cux1EIyvYPncucnjpwoOdbnMp3gBxYowM8nXQb55MJmy0e/ydVk4y48gNc5fDPvHw4vXwxfDAd7o+/q0/3I4Ou8P8pt0MpT1W1D2y7M4uVZTsaAt81X8dhY/ngnitiba4Hvysv98eAXInl28PrsaXvx4cXJZwrVHlEuYO83VqTzetYGVzby8fJnc1BOYULj6alGjeCXY3u4aCz/DwxrsJSp1UeZZhY+Uo0CAtRT5S6puaWZOXfwpjsJa25ZsWeMu1aBXYlIz/JaIpUhVsl+raKRczbzYqpzTDa6+2ZhWm1LKMccu83jWHn5ccejSJB0w8ZK7AGK/u5vrhCbox56l3qcPlTZd/4m4C0Y4TuWi5Dh1J2z9QtC6+392j0r3P+we3N//cCdJayrBghmeBAwiXBpuf/2M7NndPWsXFMiARmpZwwNN9j3MmPRakf3cS64WvkR+ESmSfjouI+UvKKwsMQY/YC9/16BaeDrn7nEmZt9IMfvEVqA2nV+amR64/z+EejHJh7J7f9RFnY8tuffpfaRN9X88ps77Pf+/u7//tKj/u4f3v/9zN6m4/i/5S/N34pYvAVB3AF1wn72DTv5E1/v1bWUWHsN9jrcA6HvNcpcmlC4EWNqGtIi+Qgvj7sDmjnXuRB5hsOQNAfLhFtX8OzW0KFr42lG7oI0QULfmmSd1HrSMVd1ORxkpfctClruMRPtxuwy83S6DyUz6lmSsuueiglXyvguT6Wb3XdQ0qoIl84RaUK9MsrqRVisxMfeZKGaM1SEUlb8oUXi1IbLqTL7PqlhdxY22miaLtvrG8DCv/LKWtbjfWAVwsc9chPOu+Ducx0CW1RSXg75AuFyDKF1kIV7kXVsx+dllHxZptJxoVWm6bMl1Gf0ckZCEZVZU3GZSwZnFBSXZ8ybqpNSsmgmU26mYaNbzYxPdlZeNdzJ++vjXn9AGQuf16q5h5gratcw1OHm/YTNKiVTG/0i7ay6ozm9vfZAJ4cf2zO4mber/m+XOTaYAa/z/7sFu8f6vp/vd+/v/7iQ1jhiovAFQD8MPvjRTChDJ7aLXBtngLSgW9X155a+lIhUsYDZaEktvslvyh8x7ZOcfb9v42O7VRyk8aZu7ONq99uXgvP37P3caI5P10zKLObrV3GoONN4uXEmW3yWDVhXQ9u+btAyG1JKGJmt5sfkLZPAwpGhppkDIF+Ty7Pisp37NVdZFGxFzCq4O/j4o3kDjyGNBGGUacuLzcMJifZGH+bHW6xTvd7HJBQv4TP7CZ4AnHwXH3xQVmYRk+/V/n+3ZRwf4CypkzFiIPySPsqDjUxtzXDkUVrnbMrKiShi2gv/+BOgT9QLMgqWKZdPG/eQcchzz3COnPEToikjZxE1u/yJlZMMqPKiebKqQphwI5QDbWF4Svfvl2v+UzbEt7b++8qKZG7D+/u+i/T/A1/f2/w5ScQpUNO3Gw/60Nl/uTAnK66+QQMvkuuQ3DC9Up75gf2z9sOn4n0V00+X/tfu/3afF33/pHtzH/99NKvj/yF79C5kFn7+Noi4ccBjBQ8pWRXD7ttvWKuMKD3NBiXPu9nXhyh8JaCRzFqDSUHsYT6SiL8aVy0eOLucplfP/7V3db9MwEH/3X2E1D5PQ4iXdhoqfKFtAFaNMXYWEEA/dmrVBLJ2adCz/PT6fv5K4apAQIJR7qDbX58SJe+ePu9/PhLvKwsyNGjJfYRDQ0QubUvWQ5WOcBrqhOA9ivrut9vB2tm+K2VYYqvoJOztptokScY5rgvF8uR5Q3sr3sEFFXa0n7q+5TxVLMK7Imd9BT9zKzNb7Z3HS/1fpZv8V/OqvGn4lB+z/6XnczP+Khz3/y58RjM+XVlFzg3Ka7tjqbgsW3gwPsf6CLVhTcOKeJvqsQblYcYv7++iE9k/up5vyGjFsCamHN3PHPkOEM9dnJCQQS74SwHIRoFGtERW+UqE3qm4rKhf4NgGFqJrQcn1Xy8SPBgYyStW1uLlZ7mRp6fQOoteso2gUEWIPLiCrlAjrZq+1dC7S9jtBy4cENQ8SD0cfsoOWdQDOdkCIE8gOlRqpAJyei8K9Ae56sgpzZ5NOYHxjuyFzyMQxgN8X3c/paUTFfTW5ebn7zL304rHMY6mxKXN1GU1wjg8WH6ssMi9RJ0dvNO5mqOA9N/fIPI/Y4cu8UNDhHl+HvtJBbVDE4N8261xopa/T54WYEktsLlVDJ/xoQGTQDp+iIRODjX1PSzHc77bVY8k229XJUvh22L6o1H6bjxD7ZVTATtz8Sg74y+lNFLvQPkbTYacW/39UwLYScRAUr9LyqKAJXlxjVWptMcEBeZO8m0wpfs5uxvR6Nvk0nif0ffJZfm9qM9bQTKaXfg14x15KagV8dgyfZ+xMVmzSRUNHNEKvOoQHjAlnJBX0xzrN6Q5IEAw6mAZMfsqW8hUGte5dJLP55O3kQtym7lSgexQ43fHV+32NEUqbLPaOsZA/QtGAmMbC8D7GZDCJoooAG5U+dNW7rLIxG1NCDnFbu43jZQsEcw5DOWKtdrioE2eDLfRT7hHSTszh9MtXsNq+2HfgzoERhJkXXP5dC8+3fH0ELSJazpnjo1ZZud7dwu/P+iS/bkfPtq9v2Ih0Z4OIDdkrYWq1MieGX4NDYMTf9ua99NJLL7300k1+AkwOWfAAoAAA
```

Reply:
How did you find out that it is the tar.
On my Mac:
```console
$ tar --version
tar (GNU tar) 1.34
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
```

On golang:1.20.5 container (with which I run make generate at the end):
```console
root@f9a0617d6e02:/go# tar --version
tar (GNU tar) 1.34
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
```

Reply:
Ah good question, thanks. It's already some time ago I fixed it on my machine. It was not `tar` but `gzip`.
To fix it, do 
```bash
brew install gzip

export PATH=$(brew --prefix)/opt/gzip/bin:$PATH
```